sdk/resourcemanager/Azure.ResourceManager.Core/tests/Scenario/SubscriptionOperationsTests.cs

```diff
@@ -124,6 +124,22 @@ public async Task TestGetResourceGroupOpsLong(int length)
             }
         }
 
+        [RecordedTest]
+        public async Task TestListSubscriptions()
```

Comment:
these belong on the SubscriptionContainerTests

Reply:
Moved